tableauMaker.js

```diff
@@ -93,7 +93,7 @@ function makeTableau(candidateSet, constraintSet, options){
 			// category constraints, and a category argument was provided, 
 			// then check if the category argument is in pCat or sCat.
 			if(catExceptionConstraints.indexOf(constraint)<0){
-				if(cat && !pCat.includes(cat) && !sCat.includes(cat)){
+				if(cat && !pCat.includes(cat) && !sCat.includes(cat) &&!(cat=="any" || cat=="")){
```

Comment:
I believe "" evaluates to false, so the condition !(cat=="") should not be necessary.